process_seq.py

```diff
@@ -317,8 +345,9 @@ def extract_bcf_stats(path):
                          "number of others:",
                          "number of multiallelic sites:",
                          "number of multiallelic SNP sites:"}
-
-    stats = {}
+    p = str(path.resolve()).split("/")[-1]
```

Comment:
This path manipulation seems hacky and should probably be done in `configure_paths`. Either way, I can't tell what this is doing, it's just pulling parts of a split string so is likely quite brittle. It looks like this is pulling in a path, why can't this path be described in `configure_paths`?

Reply:
there is definitely a better way of doing this - i just basically wanted to add another entry to the stats dict, so that when writing to csv for final-stats.csv the first row of the sample-specific column was the sample name (see the resulting outputted final-stats.csv from this code)